package.json

```diff
@@ -1,93 +1,35 @@
 {
-  "name": "iframe-coordinator",
-  "version": "4.5.2",
-  "description": "Tools for coordinating embedded apps via iframes.",
-  "dependencies": {
-    "cheerio": "^1.0.0-rc.10",
-    "commander": "^2.20.3",
-    "decoders": "1.15.0",
-    "dev-cert-authority": "^1.1.1",
-    "express": "^4.17.3",
-    "find-root": "^1.1.0",
-    "http-proxy-middleware": "^1.3.1"
-  },
-  "files": [
-    "dist/client.js",
-    "dist/host.js",
-    "dist/index.js",
-    "dist/es5/client.js",
-    "dist/es5/host.js",
-    "dist/es5/index.js",
-    "dist/**/*.d.ts",
-    "cli/ifc-cli.js",
-    "cli/example-ifc.config.js",
-    "cli/embedded-app/dist/**/*"
+  "name": "iframe-coordinator-monorepo",
+  "version": "5.0.1-beta.1",
+  "workspaces": [
+    "./packages/iframe-coordinator",
+    "./packages/ifc-cli",
+    "./apps/*"
   ],
-  "main": "dist/index.js",
-  "module": "dist/index.js",
-  "types": "dist/index.d.ts",
-  "bin": {
-    "ifc-cli": "cli/ifc-cli.js"
+  "scripts": {
+    "build": "npm run build --workspaces --if-present",
+    "start-client-example": "npm run start --workspace=ifc-example-client",
+    "test": "npm test --workspaces --if-present",
+    "commit": "git-cz",
+    "lint": "commitlint -f de759fc509cf39895450dc31742cfb36b08e01b8",
+    "sync-versions": "./scripts/sync-versions.mjs",
+    "release": "standard-version --prerelease beta & npm run sync-versions",
+    "publish-libs": "npm publish --workspace=iframe-coordinator --workspace=ifc-cli",
```

Comment:
let's add `--tag=beta` here until we're confident that everything is working and publishing correctly.  That way consumers of the library won't get the 5.x release right away when they just run `npm install iframe-coordinator`.